ocs_ci/utility/azure_utils.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.compute import ComputeManagementClient
+from azure.mgmt.compute.models import DiskCreateOption
+
+logger = logging.getLogger(name=__file__)
+
+
+class AZURE:
+    """
+    wrapper for Azure
+    """
+    _compute_client = None
+    _credentials = None
+
+    def __init__(self, subscription_id, client_id, secret, tenant):
+        """
+        Constructor for Azure class
+
+        Args:
+            subscription_id (str): Subscription ID
+            client_id (str): Application (client) ID
+            secret (): Client Secret
+            tenant (str): Tenant ID
+        """
+        self._subscription_id = subscription_id
+        self._client_id = client_id
+        self._secret = secret
+        self._tenant = tenant
+
+    @property
+    def get_credentials(self):
```

Comment:
If I'm not mistaken, method name should be changed to `credentials()` as it is a property

Reply:
thanks, fixed